pkg/ebpf/write_capture.go

```diff
@@ -12,8 +12,7 @@ import (
 func (t *Tracee) processFileWrites() {
 
 	const (
-		S_IFMT uint32 = 0170000 // bit mask for the file type bit field
-
+		//S_IFMT uint32 = 0170000 // bit mask for the file type bit field
```

Comment:
Why we comment it out instead of removing? S_IFLNK, S_IFREG, S_IFBLK, and S_IFDIR are apparently not used as well.

Reply:
I want to keep this as a reference since they're fully included in Linux headers and might be used. Those bit masks are meant to walk together and always be available, but, at the same time, I didn't want static-checker to complain about them not being used. So I'll keep it as is if you don't mind.